### ISSUE TITLE
openjpeg download error

### ISSUE THREAD

**Issue by stefanalund** [nirbheek/cerbero#6]
```
Downloading http://optimate.dl.sourceforge.net/project/openjpeg.mirror/1.5.2/openjpeg-1.5.2.tar.gz
Running command 'curl -L  -k http://optimate.dl.sourceforge.net/project/openjpeg.mirror/1.5.2/openjpeg-1.5.2.tar.gz -o /Users/stefan/cerbero/sources/local/openjpeg-1.5.2/openjpeg-1.5.2.tar.gz '
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:--  0:00:46 --:--:--     0curl: (56) Recv failure: Operation timed out
Traceback (most recent call last):
  File "./cerbero-uninstalled", line 9, in <module>
    main()
  File "./cerbero/main.py", line 130, in main
    Main(sys.argv[1:])
  File "./cerbero/main.py", line 55, in __init__
    self.run_command()
  File "./cerbero/main.py", line 105, in run_command
    res = commands.run(command, self.config, self.args)
  File "./cerbero/commands/__init__.py", line 78, in run
    return _commands[command].run(config, args)
  File "./cerbero/commands/fetch.py", line 92, in run
    args.reset_rdeps, args.full_reset)
  File "./cerbero/commands/fetch.py", line 53, in fetch
    recipe.fetch()
  File "./cerbero/build/source.py", line 106, in fetch
    shell.download(self.url, self.download_path, check_cert=False)
  File "./cerbero/utils/shell.py", line 279, in download_curl
    os.remove(destination)
OSError: [Errno 2] No such file or directory: '/Users/stefan/cerbero/sources/local/openjpeg-1.5.2/openjpeg-1.5.2.tar.gz'
```

**Comment by nirbheek** [comment]
I think this happened while SourceForge was having server issues. I think this is fixed now?

**Comment by nirbheek** [comment]
openjpeg actually isn't even used by us, and was included in the gstreamer-1.0-codecs.package file accidentally. Fixed now.
